cargo-dist/src/lib.rs

```diff
@@ -117,6 +117,13 @@ fn build_manifest(cfg: &Config, dist: &DistGraph) -> DistManifest {
         })
     }
 
+    let linkage = fetch_linkage(
+        cfg.targets.clone(),
+        dist.artifacts.clone(),
+        dist.dist_dir.clone(),
+    )?;
+    let linkage = linkage.iter().map(|l| l.to_schema()).collect();
```

Comment:
hmm interesting, this doesn't make `cargo dist plan` explode?

Reply:
If there are no files to check, `fetch_linkage` is a no-op. No error.